test/language/literals/numeric/numeric-separator-literal-bil-bd-nsl-bd.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: prod-NumericLiteralSeparator
+description: `0b` | `0B` BinaryDigit NumericLiteralSeparator BinaryDigit
+info: >
+  NumericLiteralSeparator ::
+    _
+
+  BinaryIntegerLiteral ::
+    0b BinaryDigits
+    0B BinaryDigits
+
+  BinaryDigits ::
+    BinaryDigit
+    BinaryDigits BinaryDigit
+    BinaryDigits NumericLiteralSeparator BinaryDigit
+
+  BinaryDigit :: one of
+    0 1
+
+---*/
+
+assert.sameValue(0b0_1, 0b01);
```

Comment:
isn't this the same as in test/language/literals/numeric/numeric-separator-literal-bil-bd-nsl-bd-one-of.js?